test/test-helpers.js

```diff
@@ -4,26 +4,29 @@ var middleware = require('../');
 var connect = require('connect');
 var request = require('supertest');
 
-var path = '/foo';
+var path = '/foo/:fooId/bar';
```

Comment:
I modified the default test to have a path placeholder (`fooId`) so it could be tested.